.github/workflows/ci.yml

```diff
@@ -592,8 +592,11 @@ jobs:
           $RETRY $MAVEN clean install ${MAVEN_FAST_INSTALL} ${MAVEN_GIB} -am -pl "${{ matrix.modules }}"
       - name: Maven Tests
         if: >-
-          matrix.modules != 'plugin/trino-singlestore' &&
-          !((contains(matrix.modules, 'trino-delta-lake') && (contains(matrix.profile, 'cloud-tests') || contains(matrix.profile, 'gcs-tests'))) || (contains(matrix.modules, 'trino-iceberg') && contains(matrix.profile, 'cloud-tests')) || (contains(matrix.modules, 'trino-bigquery') && contains(matrix.profile, 'cloud-tests-arrow')))
+          matrix.modules != 'plugin/trino-singlestore'
```

Comment:
this should also probably have been a `contains` operation - otherwise it deserves a comment. cc: @nineinchnick

Reply:
preexisting. 
i think the idea is that ALL singlestore tests are run with secrets